package chap15_exception;

public class _02_TryCatchFilnally {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		int[] intArr = new int[3];
		
		int idx = 0;
	}

}
